pkg/tracker/psrs.go

```diff
@@ -15,6 +15,8 @@ import (
 
 var switchTime, _ = time.Parse(time.RFC3339, "2020-06-26T00:00:00+00:00")
 
+const RequestID_TRB_ETH int = 43
```

Comment:
this is exposed as a const so that can later get the granularity for the TRB to calculate the TRB price.